### ISSUE TITLE
Possible to run other routes along side or must it be standalone

### ISSUE THREAD

**Issue by cyberwombat** [leecade/koa-webpack-middleware#23]
I am trying to convert an existing HMR Express codebase and have run into all kinds of issues with the  webpack code giving 404s and others. Please take a look at the SO [post](http://stackoverflow.com/questions/39951288/unable-to-run-webpack-dev-server-hmr-with-koa2). Perhaps you have an idea as to the issue I am having.

Thank you.

**Comment by noinkling** [comment]
Yeah I noticed the `Content-Type` header always gets set to `application/octet-stream` for subsequent middleware, which prompts for download in Chrome.
